### PR TITLE
olorenchemengine

### PR DESCRIPTION
**Pull request by davidzqhuang**
Oloren ChemEngine (oce) is a software package developed and maintained by Oloren AI containing a unified API for the development and use of molecular property predictors enabling

Direct development of high-performing predictors
Integration of predictors into model interpretability, uncertainty quantification, and analysis frameworks
Here's an example of what we mean by this. In less than ten lines of code, we'll train, save, load, and predict with a gradient-boosted model with two different molecular vector representations.

```python
import olorenchemengine as oce

df = oce.ExampleDataFrame()

model = oce.BaseBoosting([
            oce.RandomForestModel(oce.DescriptastorusDescriptor("rdkit2dnormalized"), n_estimators=1000),
            oce.RandomForestModel(oce.OlorenCheckpoint("default"), n_estimators=1000)])
model.fit(df["Smiles"], df["pChEMBL Value"])

oce.save(model, "model.oce")

model2 = oce.load("model.oce")
```

y_pred = model2.predict(["CC(=O)OC1=CC=CC=C1C(=O)O"])
It's that simple! And it's just as simple to train a graph neural network, generate visualizations, and create error models. More information on features and capabilities is available in our documentation at [docs.oloren.ai](https://docs.oloren.ai/).

### CODE REVIEW

**Review by lmmentel — APPROVED** [review]
Thanks @davidzqhuang 🙌
